### ISSUE TITLE
GetInfo() path can't be read

### ISSUE THREAD

**Issue by atomGit** [evilsocket/opensnitch#1114]
check the FAQ and Known Problems: done
`opensnitchd -check-requirements`: all systems go

**Describe the bug**
if i start/restart the daemon i'm seeing this, once, in the log...

`[2m[2024-04-03 00:09:17][0m [97m[41m ERR [0m GetInfo() path can't be read`

Include the following information:
 - OpenSnitch version: 1.6.2
 - OS: Manjaro
 - Version: it's a roller
 - Window Manager: KDE
 - Kernel version: 6.6.19-1-MANJARO

**To Reproduce**
`$ sudo systemctl restart opensnitchd`

**Post error logs:** 
```
[2m[2024-04-03 00:09:17][0m [97m[104m IMP [0m Start writing logs to /var/log/opensnitchd.log
[2m[2024-04-03 00:09:17][0m [97m[41m ERR [0m GetInfo() path can't be read
[2m[2024-04-03 00:09:18][0m [97m[104m IMP [0m UI connected, dispathing queued alerts: 0
```

**Additional context**
there's this, but the version reported for opensnitch-ebpf-module is [temporarily inaccurate](https://github.com/evilsocket/opensnitch/issues/1099#issuecomment-2033143697) (it actually matches opensnitch version)

```
$ pacman -Qs opensnitch
local/opensnitch 1.6.4-1
    A GNU/Linux application firewall
local/opensnitch-ebpf-module 1.6.5-2
    eBPF process monitor module for opensnitch
```

PS: i'm liking what i'm seeing with this project and appreciate the work of the devs and contribs - thank you all

**Comment by gustavo-iniguez-goya** [comment]
Hi @atomGit ,

Change the log level to DEBUG, and see if the monitor method in use is "ebpf" , it should appear in the logs as `Process monitor method ...`

Anyway, that error would be better to be reported as Debug. Usually it's not an error, but a limitation in some scenarios.

**Comment by atomGit** [comment]
the error vanished for some reason

**Comment by gustavo-iniguez-goya** [comment]
I was about to fix this issue and I've realized that your daemon version was 1.6.2, and this is fixed on 1.6.5 :face_with_spiral_eyes: 

thank you for reporting it anyway!
